influxdb_influxql_parser/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+//! # Parse a subset of [InfluxQL]
+//!
+//! [InfluxQL]: https://docs.influxdata.com/influxdb/v1.8/query_language
+
```

Comment:
Rust doesn't allow us to set lints for the entire workspace (yet), so every crate has to do this manually. Over time we established a somewhat "stricter than default" set of lints:

```suggestion
#![deny(rustdoc::broken_intra_doc_links, rust_2018_idioms)]
#![warn(
    missing_copy_implementations,
    missing_docs,
    clippy::explicit_iter_loop,
    clippy::future_not_send,
    clippy::use_self,
    clippy::clone_on_ref_ptr
)]
```

We should probably write some meta-check at some point to ensure that all our crates use the same set of lints.